client/bindir/cloud-setup-management.in

```diff
@@ -24,20 +24,23 @@ from cloudutils.globalEnv import globalEnv
 from cloudutils.serviceConfigServer import cloudManagementConfig
 from optparse import OptionParser
 if __name__ == '__main__':
-    initLoging("@MSLOGDIR@/setupManagement.log")
+    initLoging("/var/log/cloudstack/management/setupManagement.log")
```

Comment:
Ah yes this was an unintended change. I will put it back to the original @MSLOGDIR@ tomorrow.